tensorflow/java/src/main/java/org/tensorflow/Tensor.java

```diff
@@ -170,7 +170,7 @@ public boolean booleanValue() {
   }
 
   /**
-   * Returns the value in a scalar {@link DataType#STRING} tensor.
+   * Returns the value in a scalar {@link DataType#STRING} Tensor.
```

Comment:
If it's the Java convention to use capital for classes, then I'm ok with that.

Make sure to be coherent and change it everywhere that is related to Java.